bootstrap/postgres/setup_test.go

```diff
@@ -9,8 +9,8 @@ import (
 	"testing"
 
 	"github.com/jmoiron/sqlx"
-	bootstrapRepo "github.com/mainflux/mainflux/bootstrap/postgres"
-	pgClient "github.com/mainflux/mainflux/internal/clients/postgres"
+	bootstraprepo "github.com/mainflux/mainflux/bootstrap/postgres"
+	pgclient "github.com/mainflux/mainflux/internal/clients/postgres"
```

Comment:
only one of these requires an alias, see other instances in other files where an alias is used but not required